tests/extensions/extra/tables.txt

```diff
@@ -80,3 +80,13 @@ Lists are not tables
 
  - this | should | not
  - be | a | table
+
+Add tests for issue #449
+
+Odd backtics | Even backtics
```

Comment:
> Just a note for your future PRs: you could also squash the two commits into a single one by using git commit -a --amend (or git add && git commit --amend). Sorry for not being so precise in my previous reply.

Oh.  That makes sense.  I'm clearly not a git command line warrior; I use the GitHub app to manage my workflow (and it is very limited compared to other git GUIs).  I should really play with this more and work it into my workflow.  Getting out of my comfort zone and learning git better could really help.

> Though @waylan probably uses GitHub UI for merging pull requests, and that UI has an option to squash all commits and edit the message, anyway :)

I hope so.  Hoping to avoid another rebase :).